### PR TITLE
Don 1072 remove recaptcha

### PR DESCRIPTION
**Pull request by bdsl**

### CODE REVIEW

File: src/app/login-modal/login-modal.component.ts [hunk 1]
```diff
@@ -26,21 +26,22 @@ import {PopupStandaloneComponent} from "../popup-standalone/popup-standalone.com
...
   forgotPassword = false;
   resetPasswordForm: FormGroup;
   resetPasswordSuccess: boolean|undefined = undefined;
   userAskedForResetLink = false;
-  recaptchaIdSiteKey = environment.recaptchaIdentitySiteKey;
+  protected readonly friendlyCaptchaSiteKey = environment.friendlyCaptchaSiteKey;
+  private captchaCode: string | undefined;
+  @ViewChild('frccaptcha', { static: false })
+  friendlyCaptcha: ElementRef<HTMLElement>;
+  private recaptchWidget: WidgetInstance | undefined;
```
Comment (NoelLH) [inline src/app/login-modal/login-modal.component.ts:44]: This var name seems a bit misleading?

File: src/assets/scss/_mixins.scss [hunk 2]
```diff
@@ -264,5 +264,11 @@
     #campaign-highlights-beneficiaries-and-categories-desktop {
       display: block;
     }
-  }  
+  }
+}
+
+div.frc-captcha {
```
Comment (NoelLH) [inline src/assets/scss/_mixins.scss:270]: Per discussion on previous PR, I think we should define a mixin and then import it just where needed.

**Review by NoelLH — COMMENTED** [review]
A few small things inline, everything else LGTM

File: src/app/donation-start/donation-start-form/donation-start-form.component.ts [hunk 3]
```diff
@@ -175,7 +172,7 @@ export class DonationStartFormComponent implements AfterContentChecked, AfterCon
 
   /**
    * Tracks internally whether (Person +) Donation setup is in flight. This is important to prevent duplicates, because multiple
-   * time-variable triggers including user-initiated stepper step changes and async, invisible reCAPTCHA returns can cause us
+   * time-variable triggers including user-initiated stepper step changes and async, invisible captcha returns can cause us
```
[the remark this answers is not in the document]
Comment (bdsl) [reply src/app/donation-start/donation-start-form/donation-start-form.component.ts]: Applied, thanks.

File: src/app/login-modal/login-modal.component.ts (hunk 1 above)
Comment (bdsl) [reply src/app/login-modal/login-modal.component.ts]: Right, fixed in 5ab1bb098

File: src/assets/scss/_mixins.scss (hunk 2 above)
Comment (bdsl) [reply src/assets/scss/_mixins.scss]: Thanks, done in ce7e438f

File: src/app/login-modal/login-modal.component.ts (hunk 1 above)
Comment (bdsl) [reply src/app/login-modal/login-modal.component.ts]: Thanks, fixed in 5ab1bb098

File: src/app/app.module.ts [hunk 4]
```diff
@@ -68,18 +67,6 @@ const matomoBaseUri = 'https://biggive.matomo.cloud';
...
     { provide: MAT_CHECKBOX_DEFAULT_OPTIONS, useValue: { color: 'primary' } },
     { provide: MAT_RADIO_DEFAULT_OPTIONS, useValue: { color: 'primary' } },
-    {
-      provide: RECAPTCHA_LOADER_OPTIONS,
-      useValue: {
-        // see ng-recaptcha/README.md
-        onBeforeLoad(_url: URL): {url: URL, nonce: string} {
-          return {
-            url: new URL("https://www.recaptcha.net/recaptcha/api.js"), // using this URL instead of default google.com means we avoid google.com cookies.
-            nonce: environment.recaptchaNonce,
```
[the remark this answers is not in the document]
Comment (bdsl) [reply src/app/app.module.ts:78]: Yes, done in 500a5edcdb

**Review by NoelLH — DISMISSED** [review]

**Pull request merged**